subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -1219,15 +1218,6 @@ private void validate(String output, String displayName) {
                     if (line.matches(".*use(s)? or override(s)? a deprecated API\\.")) {
                         // A javac warning, ignore
                         i++;
-                    } else if (line.contains(JAVA7_DEPRECATION_WARNING.substring(0, JAVA7_DEPRECATION_WARNING.length() - 1))) {
```

Comment:
The [relevant private methods](https://github.com/gradle/gradle/blob/a704411c1156968b702bb21a2b0aa4d43ae82894/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java#L1038) also need to be removed.